static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -302,9 +300,10 @@ const initialiseEmail = (): void => {
 };
 
 const initialiseBanner = (): void => {
+    // if (!isInVariantSynchronous(commercialIabCompliant, 'variant')) {
```

Comment:
Is this supposed to be commented out?
Maybe someone from @guardian/commercial-dev will know if test is still needed? 😊

Reply:
The `commercialIabCompliant ` test was removed in this https://github.com/guardian/frontend/pull/21910 and replaced by `consentManagementPlatformUi`

Reply:
It looks like this branch was taken off `master` a while back before the new CMP UI was integrated into the banner picker logic (https://github.com/guardian/frontend/pull/21813), bringing up to date with master is probably necessary.

Reply:
rebased this with master so the latest version has the new consentManagementPlatformUi